### ISSUE TITLE
refresh problem when I come from My account and I go to admin user and after that to admin sharedspace

### ISSUE THREAD

**Issue by PhilippeAccorsi** [tracim/tracim#1688]
## Bug description & symptom

Refresh not working from admin user to admin sharedspace if I come from My account first

## Diagnostic

*"unknown"*

## Expected
- [ ] Refresh work all the time when url is changed.

## Context and version information

- Desktop environment (if applicable)
    - *OS: debian 9*
    - *Browser firefox*

## How to reproduce

- open my account
- now go in admin user
- now go in admin workspace => url change but you stay in admin user

## Screenshots

![refresh-problem-from-my-account](https://user-images.githubusercontent.com/25848257/57301414-4509e000-70d9-11e9-826e-5529bfd0500f.gif)

## Impacted components

- *frontend*

**Comment by PhilippeAccorsi** [comment]
Refresh is also broken when you want to close folder app. When you click on cross, nothing append.

**Comment by Skylsmoi** [comment]
I have fixed the app folder not closing issue.
I have also fixed the apps opening and closing mechanism.
@PhilippeAccorsi I have tested the new mechanism properly and it has passed functional tests but I'd like you to test switching between app feature a bit more.

Open and close apps feature on every possible process like close one and reopen it, close one and open another, switch between workspaces, open on and go to admin etc ...
